_pages/mobile-interview-project.md

```diff
@@ -136,12 +136,7 @@ We've provided a README template that we'd like you to include in your source co
 
 # Submitting The Final Project
 
-Feel free to either:
-
-**1)** Zip up the project and email it to your recruiter. This can be either flat files, or a git repo containing development history.
-
-**2)** Share the Github repo with ***@sq-mobile-interview*** if you worked on Github in a private repo, and send the link to the repo to your recruiter.
-
+Please zip up the project and email it to your recruiter. This can be either flat files, or a git repo containing your development history.
```

Comment:
Maybe also "please `git clean -ddffxx` (how many of those are needed?) the repo before submitting to ensure build caches are not included"

Reply:
Hah I like to use that as signal that they're less familiar with Git as a tool but maybe that's a stretch

Reply:
Same, but when they dont its just kind of a pain in the ass

Reply:
dones 👍